escrotum/main.py

```diff
@@ -410,9 +410,17 @@ def save_file(self, pb, width, height):
         filetype = "png"
         if "." in self.filename:
             filetype = self.filename.rsplit(".", 1)[1]
+            if filetype == "jpg":
+                filetype = "jpeg"
+
+        optskeys = []
+        optsvalues = []
+        if filetype != "png":
```

Comment:
only png is not allowing quality or this is specific of jpg?

Reply:
There's various formats that accept the quality parameter, notably jpeg and webp both do. If you pass it to PNG then GDK Pixbuf produces a warning. I suppose to do it properly you'd need a switch statement on the entire list of formats, but since you're going to be saving as a PNG most of the time this isn't exactly necessary.